homework_10/FileLogger.java

```diff
@@ -0,0 +1,44 @@
+import enums.LoggingLevel;
+import exceptions.FileMaxSizeException;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger {
+    private final FileLoggerConfiguration config;
+
+    public FileLogger(FileLoggerConfiguration config) {
+        this.config = config;
+    }
+
+    public void writeInFile(String message, LoggingLevel loggingLevel) throws FileMaxSizeException, IOException {
```

Comment:
The only public methods in `FileLogger` should be `debug` and `info`, so please make `writeInFile` private